modules/@apostrophecms/asset/index.js

```diff
@@ -122,12 +128,17 @@ module.exports = {
           // just `public` and `apos`) by examining those specified as
           // targets for the various builds
           const scenes = [ ...new Set(Object.values(self.builds).map(options => options.scenes).flat()) ];
-          let bundles = [];
+          let deployFiles = [];
           for (const scene of scenes) {
-            bundles = [ ...bundles, ...merge(scene) ];
+            deployFiles = [ ...deployFiles, ...merge(scene) ];
           }
-
-          await deploy(bundles);
+          // enumerate public assets and include them in deployment if appropriate
```

Comment:
They were already copied to `/apos-build/default/modules`, now we are using glob to enumerate them from there.